ext/ITensorNetworksNextTensorOperationsExt/ITensorNetworksNextTensorOperationsExt.jl

```diff
@@ -0,0 +1,17 @@
+module ITensorNetworksNextTensorOperationsExt
+
+using BackendSelection: @Algorithm_str, Algorithm
+using ITensorBase: inds
```

Comment:
```suggestion
using NamedDimsArrays: inds
```
Defined in NamedDimsArrays now (and I think we should try to use that as much as possible instead of ITensorBase in the library to keep things more general).